packages/contentful--app-scripts/lib/utils.js

```diff
@@ -13,6 +14,32 @@ const throwValidationException = (subject, message, details) => {
   throw new TypeError(message);
 };
 
+const isValidIpAddress = (address) => {
```

Comment:
Nit: technically this method name is incorrect, is also checks fqdn's, not only ip's